src/osd/PrimaryLogPG.cc

```diff
@@ -2045,6 +2045,15 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
     }
   }
 
+  // check for op with rwordered and rebalance or localize reads
+  if ((m->has_flag(CEPH_OSD_FLAG_BALANCE_READS) || m->has_flag(CEPH_OSD_FLAG_LOCALIZE_READS)) &&
+      op->rwordered()) {
+    dout(4) << __func__ << ": rebalance reads with rwordered not allowed "
```

Comment:
minor: having only "rebalance" might be misleading. Maybe "rebelance-or-localized"?

Reply:
done!